### PR TITLE
Adding recipe for mkdocs-graphviz

### PR DESCRIPTION
**Pull request by vallsv**
Hi,

Here is a recipe for `mkdocs-graphviz` https://gitlab.com/rodrigo.schwencke/mkdocs-graphviz

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vallsv** [comment]
@conda-forge/help-python This recipe for a mkdocs extension package is ready for review.

File: recipes/mkdocs-graphviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+about:
+  home: https://gitlab.com/rodrigo.schwencke/mkdocs-graphviz.git
+  summary: Render Graphviz graphs in Mkdocs, as inline SVGs and PNGs, natively compatible with Mkdocs Light & Dark Themes, directly from your Markdown (python3 version)
+  license:  GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - rod2ik
```
Comment (chrisburr) [inline recipes/mkdocs-graphviz/meta.yaml:41]: @rod2ik, are you happy to be listed as a recipe maintainer here?

File: recipes/mkdocs-graphviz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - mkdocs_graphviz
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/rodrigo.schwencke/mkdocs-graphviz.git
+  summary: Render Graphviz graphs in Mkdocs, as inline SVGs and PNGs, natively compatible with Mkdocs Light & Dark Themes, directly from your Markdown (python3 version)
```
Comment (chrisburr) [inline recipes/mkdocs-graphviz/meta.yaml:35]: This is a bit long for a summary, it should generally be kept to less than ~80 characters.
```suggestion
  summary: Render Graphviz graphs in Mkdocs directly from your Markdown
```

File: recipes/mkdocs-graphviz/meta.yaml (hunk 1 above)
Comment (vallsv) [reply recipes/mkdocs-graphviz/meta.yaml:41]: Thanks. Sounds good. This was merged.

File: recipes/mkdocs-graphviz/meta.yaml (hunk 2 above)
Comment (vallsv) [reply recipes/mkdocs-graphviz/meta.yaml]: Thanks. Sounds good. This was merged.

File: recipes/mkdocs-graphviz/meta.yaml (hunk 1 above)
Comment (rod2ik) [reply recipes/mkdocs-graphviz/meta.yaml:41]: Hi Chris,
I am happy with being a recipe maintainer here.
Thanks,

**Pull request merged**